test/orchard/meta_test.clj

```diff
@@ -110,3 +110,14 @@
     (docs/clean-cache!)
     (testing "Including see-also is skipped"
       (is (not (contains? (m/var-meta (resolve 'clojure.set/union)) :see-also))))))
+
+(deftest ns-meta-test
+  (testing "Includes a non-nil :file"
+    (is (some-> 'orchard.test-ns-dep
```

Comment:
Why the need for `some->` here? After all you expect this not to be `nil`.

Reply:
It's the whole thread that should return something not `nil`, right? Maybe I do not follow the question...

Reply:
My bad. Too much multitasking. :D